### ISSUE TITLE
docking the terminal on the right side

### ISSUE THREAD

**Issue by juliuscrn** [bus-stop/terminus#64]
Hi,

As far I know terminus implements the possibility to move the terminal around, instead of it being fixed at the bottom of the screen. How do I dock the terminal on the right hand side of the screen?

Thanks

**Comment by UziTech** [comment]
This is currently being worked on in PR #14 

I think it works but the resizing logic needs to be fixed.

**Comment by the-j0k3r** [comment]
![generic](https://user-images.githubusercontent.com/31389848/75367493-b2813a00-58c0-11ea-8710-2b77afb623af.gif)

It works as per gif ATM

**Comment by juliuscrn** [comment]
Perfect, exactly what I needed. Thanks a lot!

**Comment by the-j0k3r** [comment]
Lets leave it open, thanks =) Glad you're happy with it.

**Comment by juliuscrn** [comment]
Yes, closing was maybe a little premature. It would be nice if it was possible to save LHS/RHS/top as the default position when the terminal starts.

**Comment by UziTech** [comment]
If you don't mind not having the views in the status-bar [`x-terminal`](https://atom.io/packages/x-terminal) is a different terminal we are working on. It is able to dock and set a default position.

**Comment by juliuscrn** [comment]
I did install and I was mildly confused. Is it a successor of terminus? What are the differences between the two?

**Comment by UziTech** [comment]
it is a successor of [`atom-xterm`](https://atom.io/packages/atom-xterm) they use different rendering engines. `x-terminal` uses [xterm.js](https://github.com/xtermjs/xterm.js) and terminus uses [term.js](https://github.com/bus-stop/term.js)

**Comment by juliuscrn** [comment]
thanks! i'll give it a try.
